cmd/alertmanager/main.go

```diff
@@ -172,7 +172,7 @@ func run() int {
 		getConcurrency = kingpin.Flag("web.get-concurrency", "Maximum number of GET requests processed concurrently. If negative or zero, the limit is GOMAXPROC or 8, whichever is larger.").Default("0").Int()
 		httpTimeout    = kingpin.Flag("web.timeout", "Timeout for HTTP requests. If negative or zero, no timeout is set.").Default("0").Duration()
 
-		clusterBindAddr = kingpin.Flag("cluster.listen-address", "Listen address for cluster.").
+		clusterBindAddr = kingpin.Flag("cluster.listen-address", "Listen address for cluster. Set to empty string if you want to disable HA mode.").
```

Comment:
I would prefer `Set to empty string to disable HA mode.`

Opinions?

Reply:
Makes sense, fixing